### ISSUE TITLE
Can't install v6.0.0-beta.3 as a dependency

### ISSUE THREAD

**Issue by ah-adarlow** [salesforce-ux/theo#117]
Running `npm install` with theo version `6.0.0-beta.3` as a dependency fails with the following error:
`npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\Users\adarlow\Documents\Design Systems\temp\test-theo\node_modules\theo\bin\theo.js'`

This can be verified with a simple package.json and a fresh install:
```json
{
  "name": "test-theo",
  "version": "0.0.1",
  "description": "Testing whether theo installs",
  "dependencies": {
    "theo": "6.0.0-beta.3"
  },
  "author": "ah-adarlow",
  "license": "UNLICENSED"
}
```

It works fine with the previous version.

**Comment by aputinski** [comment]
Sorry about that! Should be fixed in `6.0.0-beta.4`

**Comment by ah-adarlow** [comment]
Thanks for the quick turnaround.
